Wrappers/Python/ccpi/viewer/utils/colormaps.py

```diff
@@ -465,6 +465,61 @@ def get_color_transfer_function(cls, cmap, color_range):
 
         return tf
 
+    @staticmethod
+    def get_lookup_table(cmap, color_range):
+
+        tf = vtk.vtkLookupTable()
+
+
+        if not cmap in _color_map_dict.keys():
+
+            try:
+                from matplotlib import cm
+                
+                colors = []
+                for x in range(0, 255):
+                    color = cm.get_cmap(cmap)(x)
+                    colors.append([color[0], color[1], color[2]])
+            except ImportError:
+                print("To use colormaps other than: ",
+                    "{}, please install matplotlib.".format(
+                        str(list(_color_map_dict.keys()))))
+                        
+        else:
+            try:
+                colors = _color_map_dict[cmap]
+            except KeyError as e:
+                raise KeyError("Colormap: {} could not be found. \
+                     Installing matplotlib might resolve this.".format(e))
```

Comment:
```suggestion
colors = CILColorMaps.get_color_map(cmap)
```

this method does the same as what you have above. Please note my suggestion doesn't have correct indenting - needs correcting

Reply:
removed as suggested below